start.py

```diff
@@ -1,402 +1,13 @@
-    for dim in animes:
-        print("ID: \x1b[36m" + str(x) + "\x1b[0m")
-        print("TITOLO:")
-        print("\x1b[32m" + dim.find('a',attrs={'class':'badge badge-archivio badge-light'}).text + "\x1b[0m")
-        print("TRAMA:")
-        print("\x1b[37m" + dim.find('p',attrs={'class':'trama-anime-archivio text-white rounded'}).text +"\x1b[0m")
-        anime_list.append(dim.find('a')['href'])
-        print("--------")
-        x+=1
-    if(test_ID):
-        selected = 1
-    else:
-        while True: #richiedere id se + sbagliato
-            try:
-                selected = int(input("ID ('0' per uscire):"))
-                if (selected == 0):
-                    exit(0)
-                if (selected >  len(animes) or selected < 0):
-                    print("\x1b[31mCi sono solo %d risultati\x1b[0m"%len(animes))
-                    continue
-                break
-            except ValueError:
-                print("\x1b[31mNon è un ID valido, riprovare...\x1b[0m")
-    selected -=1 #la lista parte da 0
-    URL = anime_list[selected]
-    if(config["DEFAULT"].getboolean('all')):
-        get_correlati(URL)
-    else:
-        season = 1
-        selected_anime(URL)
-    if  (file_type == 0):
-        create_crawl()
-    elif  (file_type == 1):
-        downloader()
-    else:
-        sys.exit(0)
-
-def seleziona():
-    while True: #richiedere id se + sbagliato
-            try:
-                file_type = int(input("0: Crawljob 1:Standalone: "))
-                if (file_type <  0 or file_type > 1):
-                    print("\x1b[31mScelta non valida, riprovare...\x1b[0m")
-                    continue
-                break
-            except ValueError:
-                print("\x1b[31mScelta non valida, riprovare...\x1b[0m")
-    return file_type
-
+from animesaturn import AnimeSaturn
 def main():
-    global file_type
-    import_config()
-    signal.signal(signal.SIGTERM, sig_handler)
-    signal.signal(signal.SIGINT, sig_handler)
-    try:
-        file_type = int(config["DEFAULT"]['type'])
-        if(file_type == 1):
-            print("\x1b[4mUtilizzo tipologia standalone\x1b[0m")
-            print()
-        if(file_type == 0):
-            print("\x1b[4mUtilizzo tipologia crawljob\x1b[0m")
-            print()
-        if(file_type == -1):
-            file_type = seleziona()
-        elif(file_type > 1 or file_type < -1):
-            print("\x1b[31mValore inserito nel config.ini non valido\x1b[0m")
-            print("Inserire manualmente la tipologia di programma: ")
-            file_type = seleziona()
-    except ValueError:
-        print("\x1b[31mValore inserito nel config.ini non valido\x1b[0m")
-        print("Inserire manualmente la tipologia di programma: ")
-        file_type = seleziona()
-    name = input("nome:")
-    search(name)
+    animesaturn = AnimeSaturn()
+    signal.signal(signal.SIGTERM, animesaturn.sig_handler)
+    signal.signal(signal.SIGINT, animesaturn.sig_handler)
```

Comment:
```suggestion
    signal.signal(signal.SIGTERM, sig_handler)
    signal.signal(signal.SIGINT, sig_handler)
    animesaturn = AnimeSaturn()
```
il sign_alert non viene avviato se si trova dopo `animesaturn=AnimeSaturn()`